### DISCUSSION TITLE
Localization path configuration

### DISCUSSION THREAD

**Discussion by shaundr** [gatsbyjs/gatsby#38742] [Help]
The project I am working on has requirements to support multiple languages with a route structure like so:

english : /path
german: /de/path
french: /fr/path

I'm unable to find a good way to do this. The plugin https://github.com/angeloocana/gatsby-plugin-i18n forces /en/ in the path for english.

I've attempted to use a proxy and symlink and the client side routing redirects it back to its pagePath https://github.com/gatsbyjs/gatsby/blob/7f349ff7b526b12f4d390c601217dfec36dccef0/packages/gatsby/cache-dir/production-app.js#L159

**Comment by matthewraaff** [comment]
Hey,

I recommend using [this plugin](https://www.gatsbyjs.com/plugins/gatsby-plugin-react-i18next/) instead, it allows you to let any language be the default path and accessible at the main directory.

Here's an example from the demo config:
```js
...
pages: [
    {
        matchPath: '/:lang?/blog/:uid',
        getLanguageFromPath: true,
        excludeLanguages: ['es']
    },
    {
        matchPath: '/preview',
        languages: ['en']
    }
]
...
```

I hope this helps :)
